services/movie.py

```diff
@@ -15,13 +17,17 @@ def get_movies(
     if actors_ids:
         queryset = queryset.filter(actors__id__in=actors_ids)
 
+    if title:
+        queryset = queryset.filter(title__contains=title)
```

Comment:
I recommend to use "icontains" because when you look for the movie "The Maze Runner" and enter "maze" it should find the movie as well(for user's comfortability)
So:
contains - case-sensitive (when Maze and maze are different words)
**i**contains - case-**i**nsensitive (when both Maze and maze are the same)